JS_HW/task_1.js

```diff
@@ -0,0 +1,25 @@
+// Лизе нужно написать функцию sayHello(name), которая возвращает приветствие для пользователей.
+// Тем не менее, она влюблена в пользователя с именем "Mark", и хотела бы поприветствовать его немного
+// иначе. Помогите ей)
+
+// Пример вызовов:
+//    sayHello("Oleg"), //Hello, Oleg!
+//    sayHello("Vikrot"), //Hello, Vektor!
+//    sayHello("Mark"), //Hi, Mark!
+
+
+
+
+
+function sayHello(name) {
+    if (name === "Mark") {
+        console.log('Hi,' + ' ' + name + '!')
+
+        return;
+    }
+
+
+    console.log('Hello,', name + '!')
+}
```

Comment:
1. What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue
2. Your function return 'undefined'. Just because of this line: ` return;`. `return ` is responsible for given value after the function invocation. if you want to see the result of your function you have to do smth like that `console.log(sayHell('Mark'))` and inside your function should be `return 'Hi,' + ' ' + name + '!'` or `return 'Hello,', name + '!'`. Fix it here and in all places below